src/test/java/com/squareup/javapoet/AnnotationSpecTest.java

```diff
@@ -130,19 +135,96 @@ public void testHasDefaultAnnotationWithImport() {
     assertThat(file.toString()).isEqualTo(
         "package com.squareup.javapoet;\n"
             + "\n"
+            + "import java.lang.Double;\n"
+            + "import java.lang.Float;\n"
             + "import java.lang.Override;\n"
             + "\n"
             + "@AnnotationSpecTest.HasDefaultsAnnotation(\n"
             + "    o = AnnotationSpecTest.Breakfast.PANCAKES,\n"
             + "    p = 1701,\n"
             + "    f = 11.1,\n"
-            + "    m = {9, 8, 1},\n"
+            + "    m = {\n        9,\n        8,\n        1\n    },\n"
```

Comment:
Would be more readable with one logical line per physical line (and only having `\n` at end of lines)